src/main/java/org/jabref/gui/externalfiles/AutoSetFileLinksUtil.java

```diff
@@ -94,10 +98,18 @@ public LinkFilesResult linkAssociatedFiles(List<BibEntry> entries, NamedCompound
                         entry.addFile(linkedFile);
                     });
                 }
-
                 if (changed) {
                     result.addBibEntry(entry);
                 }
+                // Run Relinking Process
```

Comment:
This comment does not say anything... You can extract a method to group statements together.

The algorithm itself should be explained